### PR TITLE
Enhancement - Add more choice of file save types

### PR DESCRIPTION
**Pull request by ryanpepper**
There are many ways of saving files in Mayavi, but only png was set
to show up by as an option to the user. This pull request makes a change
to the decorated_scene.py files and expands the choice the user has
when they click to save in the GUI.

### CODE REVIEW

File: tvtk/pyface/ui/qt4/decorated_scene.py [hunk 1]
```diff
@@ -147,14 +148,26 @@ def _save_snapshot(self):
         to an image.  Note that the extension of the filename
         determines what image type is saved.  The default is PNG.
         """
+
         if self._panel is not None:
-            wildcard = "PNG images (*.png)|*.png|Determine by extension (*.*)|*.*"
+            extensions = ['*.png', '*.jpg', '*.tiff', '*.bmp', '*.ps',
+                          '*.eps', '*.pdf', '*.tex', '*.rib', '*.wrl',
+                          '*.oogl', '*.vrml', '*.obj', '*.iv']
```
Comment (prabhuramachandran) [inline tvtk/pyface/ui/qt4/decorated_scene.py]: Turns out the Mayavi also supports exporting to .pov (Povray) and also .x3d (X3D) files so could you please add those too?  You might need to add that to the main menu also while you are at it.

**Comment by prabhuramachandran** [comment]
@ryanpepper -- this is great!  Thank you, can you please address the comment I have made.  I'll merge it after that.  Thanks!

**Comment by ryanpepper** [comment]
@prabhuramachandran Hi Prabhu, I've tried to do the things you suggested now - is there anything else?

**Comment by prabhuramachandran** [comment]
@ryanpepper -- thanks, can you please not make any whitespace changes on the PR as this makes it difficult to review and some of the changes are not relevant to the PR.  Some of the changes are also conventions that have been followed, so please do not change these.  Is it possible to fix those?

File: tvtk/plugins/scene/ui/actions.py [hunk 2]
```diff
@@ -13,9 +13,8 @@
 
 
 class SceneAction(Action):
-    """ Base class for actions that require a scene manager. """
 
-    #### 'SceneAction' interface ##############################################
```
Comment (prabhuramachandran) [inline tvtk/plugins/scene/ui/actions.py]: Please do not remove these comments.

**Pull request closed without merging**